### PR TITLE
Move every occurrence of MeiliSearch to Meilisearch

### LINKED ISSUE #215: Case mismatch between loaded and declared class names: "MeiliSearch\Client" vs "Meilisearch\Client".
**Description**
See latest release of https://github.com/meilisearch/meilisearch-php/releases/tag/v0.26.1

**Environment (please complete the following information):**
- Meilisearch version: 0.26.1
- meilisearch-symfony version: 0.9
- symfony 6.2.2

grafik

### PR DESCRIPTION
**Pull request by brunoocasali**
Move every occurrence of `MeiliSearch` to `Meilisearch`, files included.
Breaking change in the `meili_search.yaml` to `meilisearch.yml` and the root key from `meili_search` to `meilisearch`.

Also, fixes #215

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "php": "^7.4|^8.0",
     "ext-json": "*",
     "doctrine/doctrine-bundle": "^2.4",
-    "meilisearch/meilisearch-php": "^0.26.0",
+    "meilisearch/meilisearch-php": "^0.26.1",
```
Comment (mmachatschek) [inline composer.json:24]: Good 👍

File: src/Command/MeiliSearchClearCommand.php [hunk 2]
```diff
@@ -2,16 +2,16 @@
 
 declare(strict_types=1);
 
-namespace MeiliSearch\Bundle\Command;
+namespace Meilisearch\Bundle\Command;
```
Comment (mmachatschek) [inline src/Command/MeiliSearchClearCommand.php:5]: The casing in the filenames of this class and the other ones in this folder should be changed too I suppose 🤔

**Review by mmachatschek — CHANGES_REQUESTED** [review]

File: src/Command/MeiliSearchClearCommand.php (hunk 2 above)
Comment (brunoocasali) [reply src/Command/MeiliSearchClearCommand.php:5]: Yeah, I had an amazing time with git caching now, I cloned the repo twice and none of my file renames were taking place... anyway, it is fixed now!

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "meilisearch/search-bundle",
+  "name": "meilisearch-local/search-bundle",
```
Comment (mmachatschek) [inline composer.json]: This looks wrong 🤔

**Review by mmachatschek — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (brunoocasali) [reply composer.json]: Github is messing with me today 😠. I'm trying to test how breaking my changes are but it is not easy today!

Comment (brunoocasali) [reply composer.json]: I have a simple symfony app locally and when I try to install this is happening:

```bash
Generating optimized autoload files
62 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  Symfony\Component\ErrorHandler\Error\ClassNotFoundError {#64
!!    #message: """
!!      Attempted to load class "MeiliSearchBundle" from namespace "MeiliSearch\Bundle".\n
!!      Did you forget a "use" statement for another namespace?
!!      """
!!    #code: 0
!!    #file: "./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php"
!!    #line: 94
!!    trace: {
!!      ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:94 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:386 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:786 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:128 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:168 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:74 { …}
!!      ./vendor/symfony/console/Application.php:166 { …}
!!      ./bin/console:43 {
!!        › $application = new Application($kernel);
!!        › $application->run($input);
!!        › 
!!        arguments: {
!!          $input: Symfony\Component\Console\Input\ArgvInput {#3 …}
!!        }
!!      }
!!    }
!!  }
!!  2022-12-19T21:26:57+00:00 [critical] Uncaught Error: Class "MeiliSearch\Bundle\MeiliSearchBundle" not found
!!  
Script @auto-scripts was called via post-update-cmd
```

Comment (brunoocasali) [reply composer.json]: Now I can ensure all the files loaded by my app are the same as this branch without caching issues.

Comment (brunoocasali) [reply composer.json]: Fixed :)

**Review by mmachatschek — APPROVED** [review]
LGTM 👍

**Review by alallema — APPROVED** [review]
Good job !!! 😅 
LGTM!

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**